### PR TITLE
filters go into a file with a .py extension

### PR DESCRIPTION
**Pull request by dlangille**
That file is defined by the functions directive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2673,7 +2672,7 @@ for a more sensible example.
 
 A notification can be filtered (or suppressed) using a custom function.
 
-An optional `filter` in our configuration file, defines the name of a function we provide, also in the configuration file, which accomplishes that.
+An optional `filter` in our configuration file, defines the name of a function we provide. The function is provided in an other file, as specified by the functions directive in the configuration file.
```
Comment (sumnerboy12) [inline README.md]: Minor typo; `an other` -> `another`.

Comment (dlangille) [reply README.md]: Will fix.

Comment (sumnerboy12) [reply README.md]: Will merge once fixed - thanks!

**Comment by sumnerboy12** [comment]
Merged - thanks for contributing!

**Pull request merged**